addon/authenticators/oauth2-password-grant.js

```diff
@@ -31,6 +32,18 @@ const keys = Object.keys || emberKeys; // Ember.keys deprecated in 1.13
   @public
 */
 export default BaseAuthenticator.extend({
+  init() {
+    this._super(...arguments);
+    deprecate(`Ember Simple Auth: Client ID as Authorization Header is deprecated in favour of Client ID as Query String Parameter.`,
```

Comment:
This should not actually be in the `init` as using the authenticator as such is totally fine - just leaving `sendClientIdAsQueryParam` as `false` is deprecated.

Reply:
Done. Moved the `authenticate`, triggers only if set to false.